src/daemon.rs

```diff
@@ -1,3 +1,32 @@
+//! The daemon handles the running auto clock speed instance
+//!
+//! # Modes
+//!
+//! The auto clock speed daemon has two different modes
+//! - **Edit mode**
+//!     - Modifies the system cpu governor based on information such as battery state and cpu usage
+//!     - Requires sudo to run
+//! - **Monitor Mode**
+//!     - Displays information about the system to the user
+//!     - Runs in without sudo
+//!
+//! The selected mode is passed to the daemon through the settings object
+//!
+//! # Updating
+//!
+//! Data within the daemon struct gets updated every `daemon.settings.delay` millis or every
+//! `daemon.settings.delay_battery` millis when on battery.
+//!
+//! The data gets updated in the `update_all` method that gets called periodically from the `run`
+//! method.
+//!
+//! # Extra Features
+//!
+//! When not disabled the by user, the daemon will print out pretty printed data to stdout. The creation of this
+//! print string is controlled by `preprint_render` and `postprint_render`.
+//!
+//! When enabled by the user the daemon will log all of the computer telemetry to a csv file.
```

Comment:
Maybe computer cpu stats in a better way to describe the data. It has no identifiable information.

Reply:
"log all of the computer's cpu stats to a csv file"

Reply:
te·lem·e·try
/təˈlemətrē/
noun: telemetry

    the process of recording and transmitting the readings of an instrument.
    "measurements will be transferred by radio telemetry to the shore station"

Telemetry is a good word to describe the logging of data so I think we should consider keeping it.
Maybe instead of "computer telemetry" we change it to "daemon telemetry" just to make the distinction.

Reply:
If you list all of the data that is logged in the comment, then I think it is fine.

Reply:
A lot of the time when the word telemetry is used, it refers to personally identifiable information however this use does not include personally identifiable information and I want to make sure the comments reflect that. I do really appreciate the PR.

Reply:
I think telemetry implies that the data is sent somewhere else or to a remote location and I think it's important we make sure people know that no data leaves the local machine.